vent/api/plugin_helpers.py

```diff
@@ -244,14 +244,24 @@ def start_sections(self,
                 self.logger.info(status)
                 chdir(cwd)
 
-            # set docker settings for container
-            manifest = Template(self.manifest)
-            status = manifest.option(section, 'docker')
-            self.logger.info(status)
             tool_d[c_name] = {'image': image_name,
                               'name': c_name}
-            if status[0]:
-                options_dict = json.loads(status[1])
+            # get rid of all commented sections in various runtime
+            # configurations
+            manifest = Template(self.manifest)
+            overall_dict = {}
+            for setting in ['info', 'docker', 'gpu', 'settings', 'service']:
```

Comment:
why not grab the section names from the template rather than hardcoding them here?

Reply:
I did it that way because those are the only sections that a user can comment (i.e. the only ones they can configure through the vent interface), so I didn't want to look through every section in the manifest when I already knew which subset we were looking for. Also, those are currently the only sections vent looks for in the vent.template when evaluating tools I believe.